cookbooks/aws-parallelcluster-common/resources/nfs/nfs_alinux2.rb

```diff
@@ -17,10 +17,20 @@
 unified_mode true
 
 use 'partial/_install_nfs4_and_disable'
+use 'partial/_configure'
 
 default_action :setup
 
 action :setup do
   action_install_nfs4
   action_disable_start_at_boot
 end
+
+action_class do
+  def override_server_template
+    edit_resource(:template, node['nfs']['config']['server_template']) do
+      source 'nfs/nfs.conf.erb'
+      cookbook 'aws-parallelcluster-config'
```

Comment:
Since the resource has been moved to `aws-parallelcluster-common`, shouldn't we move the template there too?